playbooks/pulp.yml

```diff
@@ -23,6 +23,7 @@
           CONN_MAX_AGE: 0
           OPTIONS:
             sslmode: '{{ postgres_sslmode }}'
+      static_root: /var/lib/operator/static/
```

Comment:
@fao89 Is this a setting pulp/galaxy_ng can currently consume? or would this need to be added?

Reply:
I don't understand

Reply:
My mistake, I didn't look at the context, this is not pulp_settings, this is just an ansible var to be used by the other tasks in this PR.